scripts/globals/abilities/repair.lua

```diff
@@ -0,0 +1,131 @@
+-----------------------------------
+-- Ability: Repair
+-- Uses oil to restore pet's HP.
+-- Obtained: Puppetmaster Level 15
+-- Recast Time: 3:00
+-- Duration: Instant
+-----------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/pets");
+
+-----------------------------------
+-- onAbilityCheck
+-----------------------------------
+
+function onAbilityCheck(player,target,ability)
+    if (player:getPet() == nil) then
+        return MSGBASIC_REQUIRES_A_PET,0;
+    else
+        local id = player:getEquipID(SLOT_AMMO);
+        if(id >= 18731 and id <= 18733 or id == 19185) then
```

Comment:
```lua
        if ((id >= 18731 and id <= 18733) or id == 19185) then
```

Makes it clearer what you are doing and avoids future probs when the next editor adds any new item SE creates that would get added here..